src/Evolution/DgSubcell/Tags/Mesh.hpp

```diff
@@ -0,0 +1,21 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+
+/// \cond
+template <size_t Dim>
+class Mesh;
+/// \endcond
+
+namespace evolution::dg::subcell::Tags {
+/// The mesh on the subcells
```

Comment:
I couldn't find this documentation; it appears the Tags namespace in subcell is not documented?

Reply:
okay, I found the documentation for the Mesh tag; but the Tags namespace does not appear to be documented